CSLabs.Api/Controllers/ModuleController.cs

```diff
@@ -73,6 +73,29 @@ public async Task<IActionResult> GetForModuleEditor(string uuid)
             return Ok(module);
         }
 
+        // GET api/module/modules-editor
+        [HttpGet("modules-editor")]
+        public async Task<IActionResult> GetUsersModules()
+        {
+            if (!GetUser().CanEditModules()) {
+                return Forbid("You are not allowed to edit modules");
+            }
+            
+            var module = await this.DatabaseContext
```

Comment:
You are making 2 queries if the user is an admin. Return the modules in the if statement if the user is admin before this query to reduce the queries down to 1.